lib/re_glob.mli

```diff
@@ -26,7 +26,7 @@ exception Parse_error
 
 val glob :
   ?anchored:bool ->
-  ?period:bool ->
```

Comment:
I think this is true? `?period` was added since the version was bumped to 1.4.1.